projects/storefrontlib/src/cms-components/checkout/services/checkout-details.service.ts

```diff
@@ -19,9 +26,10 @@ export class CheckoutDetailsService {
     private checkoutService: CheckoutService,
     private cartService: CartService
   ) {
-    this.cartId$ = this.cartService
-      .getActive()
-      .pipe(map(cartData => cartData.code));
+    this.cartId$ = this.cartService.getActive().pipe(
+      map(cartData => cartData.code),
+      filter(Boolean)
```

Comment:
May be better to use: filter(cartData => !!cartData).

Makes it clearer and we don't lose type inference.

https://github.com/Reactive-Extensions/RxJS/issues/948#issuecomment-279696051